SEED-platform/seed
# #1369 - Add badges to each dq admin tab to indicate number of rules

#### Any background context you want to provide?
#### What's this PR do?
Added badges to property and taxlots dq tabs to indicate number of rules on page.

#### How should this be manually tested?
1. Navigate to data quality admin page. Should see a badge indicating the correct number of rules;
2. Create a new rule, badge number should increase by 1;
3. Create a new rule without a field and hit 'save' button, number does not change;
4. Delete a rule, badge number should decrease by 1.

What are the relevant tickets?
#1369

#### Screenshots (if appropriate)
<img width="1353" alt="Screen Shot 2020-04-30 at 7 19 29 PM" src="https://user-images.githubusercontent.com/49968203/80815303-a0ef3380-8b8a-11ea-97b1-1a57843ee905.png">
<img width="1352" alt="Screen Shot 2020-04-30 at 7 19 49 PM" src="https://user-images.githubusercontent.com/49968203/80815308-a2206080-8b8a-11ea-8a3c-ecbe00cbb562.png">
